### DISCUSSION TITLE
Ignore lsp_progress from null-ls source

### DISCUSSION THREAD

**Discussion by FotiadisM** [folke/noice.nvim#430] [Q&A] [answered]
Hello everyone,
Is there a way to disable lsp progress events from the null-ls lsp server? On `CursorMoved` event  it triggers some sort of action with my current config which generates lsp progress event, and thus, a lot of noise

**Comment by yutkat** [comment, marked as answer]
I had the same problem before and investigated it, but now I think it's impossible.

You can use fidget.nvim because it allows you to set filters.
https://github.com/j-hui/fidget.nvim
